### PR TITLE
Fixed typos in API

### PR DESCRIPTION
**Pull request by victorjdg**
Signed-off-by: Víctor Javier Díaz Garrido <vicdigar@hotmail.com>

### CODE REVIEW

File: devel/api.md [hunk 1]
```diff
@@ -771,32 +775,32 @@ Example:
...
+                LinuxBridge.PORT_SUBTREE: [
+                    {
+                        LinuxBridge.Port.NAME: "eth1",
+                        LinuxBridge.Port.STP_PRIORITY: 32,
+                        LinuxBridge.Port.STP_HAIRPIN_MODE: False,
+                        LinuxBridge.Port.STP_PATH_COST: 100
+                    }
+                ]
+            }
+        }]
```
Comment (ffmancera) [inline devel/api.md]: These extra tabs are wrong. Please remove them. As you can see, each indentation is one tab size. Thanks!

**Review by ffmancera — CHANGES_REQUESTED** [review]
Thanks for working on this. I have one comment below. If have any questions, please let me know :)

**Review by ffmancera — APPROVED** [review]
Looks good, thanks! LGTM.

File: devel/api.md (hunk 1 above)
Comment (victorjdg) [reply devel/api.md]: Done! Thank you :)

**Pull request merged**